pkg/release/major.go

```diff
@@ -98,6 +98,10 @@ func mustFindMajors(version string, releases []v1alpha1.Release) (v1alpha1.Relea
 func mustFindLatestMajor(version string, releases []v1alpha1.Release) v1alpha1.Release {
 	vv := mustToSemver(version)
 
+	if vv.PreRelease == "dev" {
```

Comment:
This is really ugly because it means nobody can test releases which have the `-dev` suffix. We maybe want to talk about that.

Reply:
We quickly discussed after standup. This is "fine".